packages/amplify-e2e-tests/src/__tests__/general-config/general-config-headless-init.test.ts

```diff
@@ -0,0 +1,36 @@
+/**
+/**
+ * Tests for headless init/pull workflows on git-cloned projects
+ * These tests exercise workflows that hosting executes during backend builds
```

Comment:
does split e2e script generate a job for this anyway ?

Reply:
https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/17503/workflows/201543cd-361d-4da5-9b62-fe15cf30a791/jobs/850035

Reply:
this is ok https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/17503/workflows/201543cd-361d-4da5-9b62-fe15cf30a791/jobs/850009 ?

Reply:
yeah this one will work since we are setting profile and this work as we will set the AWS_SDK_LOAD_CONFIG as per above comment

Reply:
https://github.com/aws-amplify/amplify-cli/blob/e927af263da7b0eb569bf1e192e435ad436d54cb/packages/amplify-provider-awscloudformation/src/aws-utils/aws.js#L13 , this line sets the env variable


Here we define a profile even if general config  profile is set : https://github.com/aws-amplify/amplify-cli/blob/e9e0d157dc60c3715b25966952083e0c428cc6d2/packages/amplify-provider-awscloudformation/src/configuration-manager.ts#L691

Reply:
sounds good.

Reply:
one case covers the case where you  have .aws (npm run setup-profile) folder and other without .aws folder